client/cypress/integration/Navigating_spec.js

```diff
@@ -0,0 +1,64 @@
+describe("Navigate the App", () => {
+  it("creates a user sent to the DB and sends the user to the log in page", () => {
```

Comment:
You do not assert "and sends the user to the log in page".

Maybe add this to the end of your test:
```js
cy.url().should("include", "/login");
```